### PR TITLE
chore(deps): update package dependencies

### PR DESCRIPTION
**Pull request by muhabdulkadir**
# Closes #<CORRESPONDING ISSUE NUMBER>

### Changes
- Updates dependencies

### Related Issues
- Issue #<NUMBER>
- Pull Request #<NUMBER>

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/main/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `main` from `fork:branchname`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -127,5 +127,12 @@
         "file": "HEADER"
       }
     }
+  },
+  "dependencies": {
+    "@accordproject/concerto-cto": "3.19.2",
+    "axios": "1.7.7",
+    "jsome": "2.3.26",
+    "pdfjs-dist": "4.7.76",
+    "webpack": "5.95.0"
```
Comment (jamieshorten) [inline package.json:136]: Why have these dependencies been added?

File: packages/markdown-cicero/package.json [hunk 2]
```diff
@@ -51,10 +51,16 @@
...
+    "@accordproject/concerto-core": "3.19.2",
+    "@accordproject/concerto-cto": "3.16.1",
     "@accordproject/markdown-common": "*",
     "@accordproject/markdown-it-cicero": "*",
+    "axios": "1.7.7",
+    "jsome": "2.3.26",
     "markdown-it": "^13.0.1",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3",
+    "webpack": "5.95.0",
```
Comment (jamieshorten) [inline packages/markdown-cicero/package.json:63]: Why do we need all these new dependencies?

File: packages/markdown-cli/package.json [hunk 3]
```diff
@@ -54,9 +54,14 @@
...
-    "@accordproject/concerto-util": "3.16.10",
+    "@accordproject/concerto-cto": "^3.19.2",
+    "@accordproject/concerto-util": "3.19.2",
     "@accordproject/markdown-transform": "*",
-    "jsome": "2.5.0",
+    "axios": "1.7.7",
+    "jsome": "^2.3.26",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3",
+    "webpack": "5.95.0",
```
Comment (jamieshorten) [inline packages/markdown-cli/package.json:64]: Why are there so many new dependencies here? Also looks like `jsome` has been downgraded.

File: packages/markdown-common/package.json [hunk 4]
```diff
@@ -51,9 +51,15 @@
...
-    "@accordproject/concerto-core": "3.16.10",
+    "@accordproject/concerto-core": "3.19.2",
+    "@accordproject/concerto-cto": "3.19.2",
     "@xmldom/xmldom": "^0.8.1",
+    "axios": "1.7.7",
+    "jsome": "2.3.26",
     "markdown-it": "^13.0.1",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3",
+    "webpack": "5.95.0",
```
Comment (jamieshorten) [inline packages/markdown-common/package.json:62]: same as above

File: packages/markdown-docx/package.json [hunk 5]
```diff
@@ -68,20 +68,24 @@
     "jest": "^27.0.6",
     "jest-diff": "^27.0.6",
     "jsdoc": "^3.6.10",
-    "jsome": "2.5.0",
+    "jsome": "^2.3.26",
```
Comment (jamieshorten) [inline packages/markdown-docx/package.json:71]: Why the downgrade?

File: packages/markdown-docx/package.json [hunk 6]
```diff
@@ -68,20 +68,24 @@
...
     "webpack-cli": "4.9.1"
   },
   "dependencies": {
+    "@accordproject/concerto-cto": "3.19.2",
     "@accordproject/markdown-cicero": "*",
-    "mammoth": "^1.4.19"
+    "axios": "1.7.7",
+    "mammoth": "^1.4.19",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3"
```
Comment (jamieshorten) [inline packages/markdown-docx/package.json:88]: Why the new deps?

File: packages/markdown-html/package.json [hunk 7]
```diff
@@ -68,20 +68,24 @@
...
     "webpack-cli": "4.9.1"
   },
   "dependencies": {
+    "@accordproject/concerto-cto": "3.19.2",
     "@accordproject/markdown-cicero": "*",
     "@accordproject/markdown-common": "*",
+    "axios": "1.7.7",
     "jsdom": "^16.7.0",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3",
```
Comment (jamieshorten) [inline packages/markdown-html/package.json:88]: Why the new deps?

File: packages/markdown-pdf/package.json [hunk 8]
```diff
@@ -66,22 +66,24 @@
     "jest": "^27.0.6",
     "jest-diff": "^27.0.6",
     "jsdoc": "^3.6.10",
-    "jsome": "2.5.0",
+    "jsome": "^2.3.26",
```
Comment (jamieshorten) [inline packages/markdown-pdf/package.json:69]: Why the downgrade?

File: packages/markdown-pdf/package.json [hunk 9]
```diff
@@ -66,22 +66,24 @@
...
+    "@accordproject/concerto-cto": "3.19.2",
     "@accordproject/markdown-cicero": "*",
     "@accordproject/markdown-common": "*",
     "@accordproject/markdown-template": "*",
-    "axios": "0.23.0",
-    "pdfjs-dist": "2.13.216",
+    "axios": "^1.7.7",
+    "pdfjs-dist": "^4.7.76",
     "pdfmake": "^0.2.4",
+    "semver": "7.6.3",
```
Comment (jamieshorten) [inline packages/markdown-pdf/package.json:86]: Why the new dependency?

File: packages/markdown-template/package.json [hunk 10]
```diff
@@ -68,24 +68,27 @@
     "jest": "^27.0.6",
     "jest-diff": "^27.0.6",
     "jsdoc": "^3.6.10",
-    "jsome": "2.5.0",
+    "jsome": "^2.3.26",
```
Comment (jamieshorten) [inline packages/markdown-template/package.json:71]: Why the downgrade?

File: packages/markdown-template/package.json [hunk 11]
```diff
@@ -68,24 +68,27 @@
...
+    "@accordproject/concerto-core": "3.19.2",
+    "@accordproject/concerto-cto": "3.19.2",
     "@accordproject/markdown-cicero": "*",
     "@accordproject/markdown-common": "*",
     "@accordproject/markdown-it-template": "*",
+    "axios": "1.7.7",
     "dayjs": "1.10.8",
     "markdown-it": "^13.0.1",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3",
```
Comment (jamieshorten) [inline packages/markdown-template/package.json:91]: Why the new deps?

File: packages/markdown-transform/package.json [hunk 12]
```diff
@@ -69,7 +69,7 @@
     "jest": "^27.0.6",
     "jest-diff": "^27.0.6",
     "jsdoc": "^3.6.10",
-    "jsome": "2.5.0",
+    "jsome": "^2.3.26",
```
Comment (jamieshorten) [inline packages/markdown-transform/package.json:72]: Why the downgrade?

File: packages/markdown-transform/package.json [hunk 13]
```diff
@@ -92,7 +92,9 @@
     "@accordproject/markdown-template": "*",
     "dijkstrajs": "^1.0.2",
     "html-to-docx": "^1.8.0",
-    "jszip": "^3.10.1"
+    "jszip": "^3.10.1",
+    "pdfjs-dist": "4.7.76",
+    "semver": "7.6.3"
```
Comment (jamieshorten) [inline packages/markdown-transform/package.json:97]: Why the new deps?

**Review by jamieshorten — COMMENTED** [review]
I see a lot of new deps and some cases of apparent downgrades.

File: package.json (hunk 1 above)
Comment (muhabdulkadir) [reply package.json:136]: I ran `npm audit fix`

**Review by ekarademir — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jamieshorten) [reply package.json:136]: We shouldn't need to add any new deps, only update some of them. There should be no downgrades either.

**Review by jamieshorten — APPROVED** [review]

**Pull request merged**